<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicsEffectClass</class>
 <widget class="QWidget" name="GraphicsEffectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraphicsEffect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QLabel
{
font-size: 14px;
font-family: 微软雅黑;
color: #000000;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_blur">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>192</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/GraphicsEffect/qrc/1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_blur">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>610</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>610</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>模糊</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>610</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>透明</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_opacity">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>610</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>660</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>颜色</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_colorize">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>660</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>158</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>模糊</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_opacity">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>50</y>
        <width>192</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/GraphicsEffect/qrc/2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>透明</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_colorize">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>330</y>
        <width>192</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/GraphicsEffect/qrc/3.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>290</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>颜色</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_dropShadow">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>660</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>660</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>阴影</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_dropShadow">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>330</y>
        <width>192</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/GraphicsEffect/qrc/4.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>290</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>阴影</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GraphicsEffect.qrc"/>
 </resources>
 <connections/>
</ui>
